samples/src/JustSaying.Sample.Restaurant.KitchenConsole/Program.cs

```diff
@@ -1,83 +1,109 @@
 using System;
 using System.Threading.Tasks;
+using JustSaying.Sample.Restaurant.KitchenConsole.Handlers;
 using JustSaying.Sample.Restaurant.Models;
 using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-using Microsoft.Extensions.Logging;
+using Serilog;
 
 namespace JustSaying.Sample.Restaurant.KitchenConsole
 {
     internal class Program
     {
         public static async Task Main()
         {
+            Log.Logger = new LoggerConfiguration()
+                .WriteTo.Seq("http://localhost:5341")
```

Comment:
Have some thoughts here: https://github.com/justeat/JustSaying/pull/655#discussion_r392254600. 

It would be great to have it for the next testing phase too, as it provides a really quick feedback cycle for running load test scenarios and seeing them graphed in Seq.